src/HackF5.UnitySpy/Detail/FieldDefinition.cs

```diff
@@ -14,13 +14,13 @@
         "Field: {" + nameof(FieldDefinition.Offset) + "} - {" + nameof(FieldDefinition.Name) + "}")]
     public class FieldDefinition : MemoryObject, IFieldDefinition
     {
-        public FieldDefinition([NotNull] TypeDefinition declaringType, uint address)
+        public FieldDefinition([NotNull] TypeDefinition declaringType, IntPtr address)
             : base((declaringType ?? throw new ArgumentNullException(nameof(declaringType))).Image, address)
         {
             this.DeclaringType = declaringType;
             this.TypeInfo = new TypeInfo(declaringType.Image, this.ReadPtr(0x0));
-            this.Name = this.ReadString(0x4);
-            this.Offset = this.ReadInt32(0xc);
+            this.Name = this.ReadString(this.Process.SizeOfPtr);
+            this.Offset = this.ReadInt32(this.Process.SizeOfPtr * 3);
```

Comment:
Would it possible to add a short explanation of where the * 3 comes from? I'm sorry I don't have a lot of knowledge about these things, so maybe it's something obvious.

Reply:
Done